client/src/components/navbar/style.css

```diff
@@ -0,0 +1,120 @@
+@import url('https://fonts.googleapis.com/css?family=Volkhov');
+body{
+    padding: 0;
+    margin: 0 auto;
+ font-size: 1.5em;
+}
+
+
+.nav{
+   
+    list-style-type: none;
+    margin: -8px;
+
+    overflow: hidden;
+    background-color: #8CA2D9;
+    margin: auto;
+    
+    
+}
+.nav img{
+    width: 50px;
+  
+  
+}
+.usernam{
+    font-family: 'Volkhov';
+    padding-right: 20px;
+} 
+
+ul {
```

Comment:
I think some of these selectors should be more specific, so for example `.nav ul { ..` so that the `ul` styles here only apply to the nav's ul, not every ul in the app

Reply:
ok

Reply:
but we under stand what it means hahaha